cypress/integration/26-register_registration/resource_types/creator_selects_resource_type_contribution_to_journal/select_resource_type_contribution_to_journal.js

```diff
@@ -107,10 +107,10 @@ Then('they see that the Peer Review Details are hidden', () => {
 // Scenario: Creator sees fields for Resource subtype "Corrigendum"
 And('they see a disabled field for Journal based on selected Journal article', () => {
   cy.get('[data-testid=article-search-field]').within(() => {
-    cy.get('input').type('Leaders');
+    cy.get('input').type('Test article corrigendum');
   });
-  cy.contains('collaborative').click({ force: true });
-  cy.get('[data-testid=article-search-field] > p').first().contains('Journal');
+  cy.contains('Test article corrigendum').click({ force: true });
+  cy.get('[data-testid=article-search-field] > p').first().contains('International Journal of Human-Computer Interaction');

```

Comment:
har prøvd å bruke mock data direkte så mye i frontend nå, slik at det blir noe slik:
```suggestion
  cy.get('[data-testid=article-search-field] > p').first().contains(mockedArticle.mainTitle);

```
mulig det ikke er like rett frem her om det er ting som ligger i aws

Reply:
Hmm, må fikse noe i test data for article her, ser jeg